scripts/bootstrap-ansible.sh

```diff
@@ -64,8 +64,19 @@ pushd ${OA_DIR}
     exit 99
   fi
 
-  # Now use GROUP_VARS of OSA and RPC
-  sed -i "s|GROUP_VARS_PATH=.*|GROUP_VARS_PATH=\"\${GROUP_VARS_PATH:-${OA_DIR}/playbooks/inventory/group_vars/:${BASE_DIR}/group_vars/:/etc/openstack_deploy/group_vars/}\"|" /usr/local/bin/openstack-ansible.rc
-  sed -i "s|HOST_VARS_PATH=.*|HOST_VARS_PATH=\"\${HOST_VARS_PATH:-${OA_DIR}/playbooks/inventory/host_vars/:${BASE_DIR}/host_vars/:/etc/openstack_deploy/host_vars/}\"|" /usr/local/bin/openstack-ansible.rc
+  # RPC-O has roles in its own git tree, so we need to add it to the
+  # path for Ansible to search.
```

Comment:
There are no roles in the rpc-openstack git tree any more.

Reply:
done.